packages/loader/container-loader/src/container.ts

```diff
@@ -946,6 +946,79 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         }
     }
 
+    public connect() {
+        if (!this.closed && !this.connected) {
+            // Note: no need to fetch ops as we do it preemptively as part of DeltaManager.attachOpHandler().
+            // If there is gap, we will learn about it once connected, but the gap should be small (if any),
+            // assuming that connect() is called quickly after initial container boot.
+            this.connectInternal({ reason: "DocumentOpenResume", fetchOpsFromStorage: false });
```

Comment:
Let's think about what the reason should be.  It used to be different for `resume` v. `setAutoReconnect`, now it will be the same.

Reply:
Thoughts on `DocumentConnect`? This would match the `DocumentOpen` reason used when the container is initially created and tries to connect.